proto/viam/app/cloudslam/v1/cloud_slam.proto

```diff
@@ -102,6 +102,8 @@ message UpdateMappingSessionMetadataByIDRequest {
   string end_status = 2;
   // set at the time of job closeout and used as the package version
   google.protobuf.Timestamp time_cloud_run_job_ended = 3;
+  // additional details on the end status if needed, such as errors
+  string end_status_msg = 4;
```

Comment:
end_status_msg -> error_msg (because if end_status is "success", end_status_msg will be empty)